configure.ac

```diff
@@ -52,8 +52,13 @@ RR_CHECK_FEATURE_LIB(TENSORFLOW, TensorFlow Installation,
 AC_LANG_PUSH([C++])
 RR_CHECK_FEATURE_LIB(TFLITE, TensorFlow lite Installation,
                      tensorflow-lite, TfLiteTypeGetName, tensorflow/lite/model.h, no)
+
+RR_CHECK_FEATURE_LIB(TENSORRT, TensorRT Installation,
+                     nvinfer, createInferBuilder_INTERNAL, NvInfer.h, no)
 AC_LANG_POP([C++])
 
+AM_COND_IF([HAVE_TENSORRT], [RR_CHECK_CUDA], [])
```

Comment:
Is this really needed?

Reply:
The TensorRT backend needs cuda headers and libraries in order to compile (it uses `cudaMalloc` & `cudaMemcpy`). This ensures that this is added for when setting up this backend